Installation/Source/openzaak.drc.source.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
Remove this file, already added this in the predesignated file for this: buren.drc.source.json